config/colcon.upstream.yaml

```
@@ -0,0 +1,6 @@
+name: colcon
+method: https://packagecloud.io/dirk-thomas/colcon
```

```suggestion
method: https://packagecloud.io/dirk-thomas/colcon/ubuntu
```

The actual repository is distribution specifc so the above will work for Ubuntu distributions but not Debian distributions suchas Buster and Stretch.

---

Is there a way to add Debian without duplicating the whole file?

---

Not that I know of in this configuration structure.

---

Split and updated the URLs / suites accordingly: 2f37b6937ff499c0fcd05360f014371f8db7d224.